pom.xml

```diff
@@ -28,6 +39,25 @@
             <scope>runtime</scope>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi-ooxml</artifactId>
+            <version>4.1.2</version>
```

Comment:
it's recommended to move all versions to the properties (where Java version is already pointed), it'll help update versions in the future and keep it all in one place. https://github.com/FedosovMax/knight-todo/blob/master/pom.xml - you can see and example here